src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -239,7 +239,8 @@ def __writeSub(self, executable, nJobs, location, processors):
 initialdir = %(initialDir)s
 grid_resource = condor %(ceName)s %(ceName)s:9619
 transfer_output_files = ""
-+xcount = %(processors)s
+machine_count = 1
```

Comment:
`machine_count` seems to depend on the `parallel` universe according to the documentation: https://htcondor.readthedocs.io/en/v8_8/man-pages/condor_submit.html

It is described in the *COMMANDS FOR PARALLEL, JAVA, and SCHEDULER UNIVERSES* section.
Using `vanilla` or `grid` universe implies exploiting `n` cpus - specified by `request_cpus` - from one machine per allocation I guess.